docs/adoption.md

```diff
@@ -17,39 +19,243 @@ Clone/fork the following repositories:
+
+```
+
+_Note: the terraform commands with their respective flags should be in a single line_
+
+### Deploy housing-repairs-online-frontend
+
+Once you have completed the above steps, add the following terraform code to main.tf to provision a static web app as this is where the frontend is deployed.
+
+```
+resource "azurerm_static_site" "hro_frontend_test" {
+  name                = "name-of-your-static-site"
+  resource_group_name = var.resource_group_name
+  location            = var.location
+}
+
+```
+
+Add the following code in your yaml file to deploy your code to the static web app:
+
+```
+  build_and_deploy_job:
+    needs: [terraform]
+    if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.action != 'closed')
+    runs-on: ubuntu-latest
+    name: Build and Deploy Job
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Build And Deploy
+        id: builddeploy
+        uses: Azure/static-web-apps-deploy@v1
+        with:
+          azure_static_web_apps_api_token: ${{secrets.AZURE_STATIC_WEB_APPS_API_TOKEN}}
+          repo_token: ${{ secrets.GITHUB_TOKEN }} # Used for Github integrations (i.e. PR comments)
+          action: "upload"
+          ###### Repository/Build Configurations - These values can be configured to match your app requirements. ######
+          # For more information regarding Static Web App workflow configurations, please visit: https://aka.ms/swaworkflowconfig
+          app_location: "/" # App source code path
+          api_location: "api" # Api source code path - optional
+          app_artifact_location: 'public' # Built app content directory - optional
+          output_location: "out" # Built app content directory - optional
+          ###### End of Repository/Build Configurations ######
+          NEXT_PUBLIC_APP_ENV: ${{ (github.event_name == 'pull_request' && 'staging') || ('production') }}
+
+
+  close_pull_request_job:
+    if: github.event_name == 'pull_request' && github.event.action == 'closed'
+    runs-on: ubuntu-latest
+    name: Close Pull Request Job
+    steps:
+      - name: Close Pull Request
+        id: closepullrequest
+        uses: Azure/static-web-apps-deploy@v1
+        with:
+          azure_static_web_apps_api_token: ${{secrets.AZURE_STATIC_WEB_APPS_API_TOKEN}}
+          action: 'close'
+```
+
+Now you have added all the resources that you need in azure in terraform, you are ready for the CI to apply the terraform and deploy. The first CI run will provision the azure static web app resource (however the deployment will fail and this is expected). Log in to the azure web portal, navigate to the static webb app you provisioned and copy the `Manage deployment token` value. Add this to github actions secret with the name `AZURE_STATIC_WEB_APPS_API_TOKEN`. As you have now added this secret, the deployment should pass successfully on the second run.
```

Comment:
Azure is a name and should always start with a capital. Please update throughout these changes.